app/Services/GraphQL.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Rogue\Services;
+
+use Softonic\GraphQL\ClientBuilder;
+
+class GraphQL
+{
+    /**
+     * Build a new GraphQL client.
+     */
+    public function __construct()
+    {
+        $this->client = ClientBuilder::build(config('services.graphql.url'));
+    }
+
+    /**
+     * Run a GraphQL query using the client.
+     *
+     * @param  $query     String
+     * @param  $variables Array
+     * @return \Softonic\GraphQL\Response
+     */
+    public function query($query, $variables)
+    {
+        return $this->client->query($query, $variables);
```

Comment:
I know you don't love this, but I for sure think it's worth it to have the config all set up properly in one place. I can't think of a more straightforward way to do it either

Reply:
Maybe down the line we'll have like `getCampaignById()` that takes an `id`, sets up the `$query` and `$variables` and calls `$this->client->query()` all for us! Something to consider once it starts getting used!

Reply:
Gotcha, and that's a sweet thought -- will def keep that in mind when setting up the queries!